evergreen/config.yml

```diff
@@ -414,12 +414,15 @@ functions:
         set -o errexit
         set -o verbose
 
-        if [[ ! -d realm-core ]]; then
+        TOP_DIR=$(pwd)/realm-core
+        REQUIREMENTS_PATH=$TOP_DIR/evergreen/hang_analyzer/requirements.txt
+        HANG_ANALYZER_PATH=$TOP_DIR/evergreen/hang_analyzer
```

Comment:
you could reverse REQUIREMENTS_PATH and HANG_ANALYZER_PATH so REQUIREMENTS_PATH=$HANG_ANALYZER_PATH/requirements.txt